spec/plugin.integration.spec.js

```diff
@@ -0,0 +1,53 @@
+var fs = require('fs');
+var path = require('path');
+
+var _ = require('lodash');
+var webpack = require('webpack');
+
+var ManifestPlugin = require('../index.js');
+
+function webpackConfig (webpackOpts, opts) {
```

Comment:
let's be consistent with style here - we should stick to `function<name>(...) {}` style as it is used all over the project

I mean - there shouldn't be a space before `(`

Also it should be fixed in other places in this PR